hack/fetch-releases.sh

```diff
@@ -120,7 +120,48 @@ release_yaml_pac() {
     echo ""
 }
 
-#Args: <target-platform> <pipelines version> <triggers version> <dashboard version> <results version> <pac version>
+
+release_yaml_hub() {
+  echo fetching '|' component: ${1} '|' version: ${2}
+  local version=$2
+
+  ko_data=${SCRIPT_DIR}/cmd/${TARGET}/operator/kodata
+  dirPath=${ko_data}/tekton-hub/${version}
```

Comment:
```suggestion
  dirPath=${ko_data}/tekton-hub
```
Currently the operator stories only 1 version of any payload. So let us make sure that kodata has only 1 set of payload for any component.

In addition, the name `dirPath` is as generic as naming a variable `x` 😛 . Make it a name that is meaningful.

Reply:
>In addition, the name dirPath is as generic as naming a variable x stuck_out_tongue . Make it a name that is meaningful.

@nikhil-thomas I also felt the same but then just saw other functions and kept it in consistent with others.

Reply:
@nikhil-thomas we would be need the ${version} since operator is going to read this version dir and display that in the TektonHub CR status
```
$ kubectl get tektonhub hub
NAME   VERSION   READY   REASON   APIURL
hub    v1.6.0    True             https://api.url